pkg/kubernetes/version.go

```diff
@@ -0,0 +1,220 @@
+package kubernetes
+
+import (
+	"fmt"
+	"sort"
+	"strconv"
+	"strings"
+
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+const (
+	// https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/
+	AnnotationSpinnakerArtifactVersion = `artifact.spinnaker.io/version`
+	AnnotationSpinnakerMonikerSequence = `moniker.spinnaker.io/sequence`
+	Annotation
+)
+
+type SpinnakerVersion struct {
+	Long  string
+	Short string
+}
+
+//Create a similar function for labels
+func (c *controller) AddSpinnakerVersionAnnotations(u *unstructured.Unstructured, version SpinnakerVersion) error {
+	annotate(u, AnnotationSpinnakerArtifactVersion, version.Long)
+	annotate(u, AnnotationSpinnakerMonikerSequence, version.Short)
+
+	var err error
```

Comment:
Move this var declaration to the top of the function before the calls to annotate.
```
	var err error

	annotate(u, AnnotationSpinnakerArtifactVersion, version.Long)
	annotate(u, AnnotationSpinnakerMonikerSequence, version.Short)
```

Reply:
Feel free to disagree with this, I just thing the var declaration makes more sense at the top of the function in this case, since it's used throughout.

Reply:
Yes it makes more sense

Reply:
done